content/governance.md

```diff
@@ -95,6 +95,11 @@ Decisions are usually made by [lazy consensus](#consensus). If no consensus can
 
 Material changes to this document are discussed publicly on the [developer mailing mailing list][devs]. Any change requires a [supermajority](#supermajority-vote) in favor. Editorial changes may be made by [lazy consensus](#consensus) unless challenged.
 
+### Editorial changes
+
+Editorial changes are changes which fix spelling or grammar.
```

Comment:
This is far more restrictive than what we have done in practice. For example someone changing company would now need a super-majority vote to update this doc.

Reply:
Valid point, will re-word.